src/api.rs

```diff
@@ -1466,6 +1513,16 @@ impl Api {
         Ok(rv)
     }
 
+    pub fn list_available_regions(&self) -> ApiResult<Vec<Region>> {
+        let resp = self.get("/users/me/regions/")?;
+        if resp.status() == 404 || resp.status == 400 {
+            return Err(ApiErrorKind::ResourceNotFound.into());
+        }
```

Comment:
If the response is a 404 should we return an empty list result? Someone could be using new sentry-cli with old self-hosted.